Makefile.am

```diff
@@ -170,7 +170,6 @@ BATS = \
 	test/functional/search/search-content-check-negative.bats \
 	test/functional/search/search-content-check-positive.bats \
 	test/functional/search/search-client-certificate.bats \
-	test/functional/search/search-experimental.bats \
```

Comment:
hmm I'm not sure why this test is being removed... maybe the rebase removed it

Reply:
Looking at all the previous commits of this PR looks like I had deleted this file since the first time I committed to this PR. I have no idea why I could have done that, so it was probably a mistake. I re-added it.